cmd/clusterctl/client/config/providers_client.go

```diff
@@ -72,6 +72,7 @@ const (
 	IonosCloudProviderName     = "ionoscloud-ionoscloud"
 	VultrProviderName          = "vultr-vultr"
 	OpenNebulaProviderName     = "opennebula"
+	ScalewayProviderName       = "scaleway"
```

Comment:
just to double check with @sbueringer 

we are ok on omitting the prefix (the github organization) because in this case prefix and provider name match.

this won't lead to name clash in the future, because if someone from another org wants to create a scaleway provider, in this case the org could not not be omitted

https://cluster-api.sigs.k8s.io/developer/providers/contracts/clusterctl#providers-github-org-prefix

Reply:
Yes exactly